### PR TITLE
Documentation FAQ Refinement

### PR DESCRIPTION
**Pull request by xinyis991105**
## Description
This PR refines the FAQ sections in the current documentation.

## Motivation and Context
To use more precise wording in the documentation and to provide more clear pointers to troubleshooting sections.

## How Has This Been Tested?
mkdocs serve and visit the local host to see changes.
In particular, the following changes have been made:
1. FAQ has been reworded Troubleshooting on each relevant page.
2. A reminder for checking out the troubleshooting section has been added for each relevant page.
No new nonspecific FAQ has been added since there are not many existing non-specific questions.
For other minor fixes, see comments. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

## Other issues / help required

If unsure, feel free to submit first and we'll help you along.

### CODE REVIEW

File: docs/lab.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ By convention, an autograder accepts an optional `-A` command line argument that
 
 One of the nice properties of Autolab autograders is that they can be written and tested offline, without requiring any interaction with Autolab. Writing autograders is not easy, but the fact that they can be developed offline allows you to develop and test them in your own familiar computing environment.
 
+To format your autoresult feedback provided to the students, use the [formatted feedback feature](/features/formatted-feedback).
```
Comment (xinyis991105) [inline docs/lab.md:28]: Moved this reminder to use formatted feedback from Troubleshooting to the feature introduction itself.

File: docs/installation/osx.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
-### Mac OSX 10.11+
```
Comment (xinyis991105) [inline docs/installation/osx.md:1]: Deleted this title since it's redundant.

File: docs/installation/docker-compose.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 The Autolab Docker Compose installation is a fast and easy production-ready installation and deployment method. It uses a MySQL database for the Autolab deployment, and comes with TLS/SSL support. This is now the preferred way of installing Autolab.
 
-If you are stuck or find issues with the installation process you can either file an issue on our Github repository, or join our Slack [here](https://autolab-slack.herokuapp.com/) and let us know and we will try our best to help. Also see the [debugging](#debugging-your-deployment) section for tips on how to diagnose problems.
+If you are stuck or find issues with the installation process you can either file an issue on our Github repository, or join our Slack [here](https://autolab-slack.herokuapp.com/) and let us know and we will try our best to help. Also see the [debugging](#debugging-your-deployment) section for tips on how to diagnose problems and check out the [troubleshooting](#troubleshooting) section if you run into any issue.
```
Comment (ashleyzhang) [inline docs/installation/docker-compose.md:5]: nit: I think "issues" instead of "issue" for the last word here

**Review by ashleyzhang — APPROVED** [review]
Minor wording suggestion but otherwise LGTM!

File: docs/installation/docker-compose.md (hunk 3 above)
Comment (xinyis991105) [reply docs/installation/docker-compose.md]: Wording fixed. I'll merge in. Thanks!

**Pull request merged**